checkfiles.py

```diff
@@ -425,6 +480,12 @@ def process_fastq_file(job, fastq_data_stream, session, url):
         '^(@SRR[\d.]+)$'
     )
 
+    pacbio_read_name_pattern = re.compile(
```

Comment:
All of these re.complie statement could be constants at the top of the file.  Then they wouldn't need to be passed around through the functions.

Reply:
fixed